### DISCUSSION TITLE
Announcements about the course

### DISCUSSION THREAD

**Discussion by SilasK** [metagenome-atlas/Tutorial_Finnland#2] [Announcements]
# Welcome to the Atlas course held in September/October 2021

**Click on the right to subscribe!** Maybe check how GitHub allows you to send notifications.

**Comment by gafnialmog** [comment]
![122effe2ad2c90ad69c9e4b6fa91e048](https://user-images.githubusercontent.com/8017020/133873764-ed9473d7-e1f0-41a9-8714-46a147159d73.png)

**Reply by tushardlodha** [reply]
Hello Everyone

**Reply by Mechah** [reply]
Hi

**Reply by mvtejesvi** [reply]
Hi

**Reply by binutripathi** [reply]
Hi everyone

**Reply by ilma74** [reply]
hello

**Reply by Felansky** [reply]
Hi everyone!!! :)

**Reply by huuskok** [reply]
Hi
